cmd/ebrelayer/main.go

```diff
@@ -129,6 +129,12 @@ func cosmosRelayerCmd() *cobra.Command {
 
 // RunEthereumRelayerCmd executes the initEthereumRelayerCmd with the provided parameters
```

Comment:
Rename RunEthereumRelayerCmd -> RunRelayerCmd as its doing both now?

Reply:
Also, are the ethereumRelayerCmd() and cosmosRelayerCmd()  cli functions still needed?

We should either have a single cli function relayerCmd(), or have 3 cli functions cosmosRelayerCmd(), ethereumRelayerCmd(), relayerCmd()

You've mixed both into ethereumRelayerCmd and left cosmosRelayerCmd hanging, right?

Reply:
https://github.com/swishlabsco/peggy/pull/11 contains the unified Relayer and is currently a separate PR that is dependent upon this PR. That PR only has relayerCmd() as recommended, while this PR primarily contains Relayer tests.

Reply:
Disregard the above, I see that you made a comment to this effect below.